textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/nlpextrator/NlpExtractor.java

```diff
@@ -188,69 +233,78 @@ public ITuple getNextTuple() throws Exception {
      * 3. The two spans have the same key (Organization, Person,... etc)
      */
     private Span mergeTwoSpans(Span previousSpan, Span currentSpan) {
-        String previousWord = previousSpan.getValue();
-        String currentWord = currentSpan.getValue();
-
-        String newWord = previousWord + " " + currentWord;
-
-        String NEConstant = previousSpan.getKey();
-        String fieldName = previousSpan.getFieldName();
-        int start = previousSpan.getStart();
-        int end = currentSpan.getEnd();
-
-        Span mergedspan = new Span(fieldName, start, end, NEConstant, newWord);
-
-        return mergedspan;
-
+        String newWord = previousSpan.getValue() + " " + currentSpan.getValue();
```

Comment:
I just merge a few lines, to make it looks concise. It does the exact same thing.